content/en/security/vulnerabilities/_index.md

```diff
@@ -16,48 +14,58 @@ further_reading:
+
+**Note**: If you're looking for vulnerability management for your application libraries and custom application code, see [Application Vulnerability Management][5].
+
+## Explore vulnerabilities
+The CSM Vulnerability Explorer shows a complete list of vulnerabilities detected across your infrastructure, ordering them based on their severity, and offering grouping and filtering capabilities so you can investigate and prioritize problems. 
 
-## Review the health of your container images
-CSM Vulnerabilities offers vulnerabilities in the container images on your hosts.
+(update screen shot when beta tags are removed)
 
-- View your container images on the [container images][2] page. 
-- Observe the number of vulnerabilities that exist in the container images (column name `VULNERABILITIES`).
+{{< img src="security/vulnerabilities/CSM_Vulnerabilities.png" alt="The CSM Vulnerability page sorting by unique vulnerabilities" width="100%">}}
 
-{{< img src="security/infrastructure_vulnerabilities/container_images_tab.png" alt="The Container Images tab highlighting vulnerabilities and container column sort feature" width="100%">}}
+## Manage container vulnerabilities
 
-View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab:
+View your container images on the [container images][2] page. 
+Observe the number of vulnerabilities that exist in the container images (column name **vulnerabilities**). Additionally, sort by **source**, **image tag**, **repo digest**, and more.
 
-{{< img src="security/infrastructure_vulnerabilities/container_vulnerability.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
+{{< img src="security/vulnerabilities/container_images_tab.png" alt="The Container Images tab highlighting vulnerabilities and container column sort feature" width="100%">}}
 
-## Identify impacted infrastructure
+View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab.
 
-- Click any vulnerability to see more information. 
-- Click **View Vulnerability in CSM** to view all related infrastructure impacted by this vulnerability.
+Select a specific vulnerability to see its details, including which containers and hosts are affected, severity breakdown score, and recommended remediation steps. On the details explorer, you can also view impacted resources in CSM to gain better insights to your overall risk.
```

Comment:
```suggestion
Select a vulnerability to see its details, including which containers and hosts are affected, severity breakdown score, and recommended remediation steps. Click **See Impacted Resources in CSM** to view the resources that are impacted by the vulnerability.
```

Reply:
If I understand correctly, you're referring to the **See Impacted Resources in CSM** button. I think we should rephrase this sentence to be centered around an action, in this clicking the button to view impacted resources.

Reply:
Yes that makes more sense thanks!